memo.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+TABLE_NAME = 'memos'
+CONN = PG::Connection.new(dbname: 'memo_app')
+
+# Memo class(index, show, create, edit, delete etc.)
+class Memo
+  def build_id
+    SecureRandom.hex
+  end
+
+  def all
+    query = "SELECT * FROM #{TABLE_NAME}"
```

Comment:
```suggestion
    query = "SELECT * FROM memos"
```

テーブル名は定数にしても特にメリットがない（むしろ定数が間に挟まる分可読性が下がる）ので、そのまま書いてしまっても大丈夫です。